docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+
+if [ "$USER_MODE" = "TEST" ];
+then
+
+    [ -d "/opt/hoover/snoop/blobs" ] && chown $UID:$GID -R /opt/hoover/snoop/blobs
+
+    chown $UID:$GID $DATA_DIR # snoop celery
+
+    echo "running as liquid"
+    exec gosu $USER_NAME "$@"
+
+else
+
+    echo "running as root"
+    exec "$@"
+
+fi;
```

Comment:
About the two different merge parts. My Idea was to give a flag as an Env variable in Drone that we now run tests and if we get the flag we run as 666 if not we run as root. Its not working like this but do is this what you wanted or did I misunderstood that?

Reply:
Not really; if the container gets run as root by nomad then permissions don't matter to it anyway, right? So you shouldn't care what happens outside of the tests right now, and you may need to add an explicit `user = root` in the liquid job for these containers